packages/DMX/0.1.0/DMX_linux.json

```diff
@@ -0,0 +1,19 @@
+{
+    "arch": "all",
```

Comment:
You'll want to specify an architecture (probably x86_64, like this https://github.com/ccrma/chugins/blob/main/Line/build-pkg-linux.ck#L36). Since it's a compiled binary. Right now we're generally shooting to support x86_64 on windows and linux, and then a universal binary on mac